### ISSUE TITLE
[BUG] Settings lost after using a new image

### ISSUE THREAD

**Issue by vdb86** [linuxserver/docker-doublecommander#19]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I had several things set up with DC, and those settings were saved, so each time the container was started the setting were loaded.
After I pulled an updated image and run the container, everything went back to default settings.
I thought the settings were saved in the /config, so after using the updated image I would have expected them to be loaded normally, but I can't find the settings file, which makes me think it is saved within the container?

### Expected Behavior

After getting an updated image and starting the new container, the settings are preserved and loaded.

### Steps To Reproduce

1. Change some settings in DC, make sure you exit so that it saves your settings
2. Start it up and verify that they are loaded
3. Now update the image and start the container again
4. You'll see that your settings are lost

### Environment

```markdown
- OS: Libreelec
- How docker service was installed: Through Libreelec
```

### CPU architecture

arm64

### Docker creation

```bash
I'm using portainer to run the containers.
---
version: "2.1"
services:
  doublecommander:
    image: lscr.io/linuxserver/doublecommander:arm64v8-latest
    container_name: doublecommander
    environment:
      - PUID=0
      - PGID=0
      - TZ=Europe/Belgrade
    volumes:
      - /storage/.kodi/userdata/addon_data/service.system.docker/docker/volumes/doublecommander:/config
      - /storage/downloads/complete:/data/complete
      - Server1:/data/server1
      - Server2:/data/server2
    network_mode: "host"
    ports:
      - 3000:3000
      - 3001:3001
    restart: unless-stopped
```

### Container logs

```bash
I can't seem to get this.
```

**Comment by aptalca** [comment]
The folder you're mapping for the config folder is meant to be managed by the docker service only. You shouldn't be bind mapping or even directly using those.

**Comment by vdb86** [comment]
Hello and thank you for your reply!
I've used the docker compose in portainer, that was provided by linuxserver.
I added my info as needed in that docker compose.
Apologies for my lack of knowledge, should have I done something in a different way?

Where could I find the double commander settings files and how do I ensure they are not lost after an updated container is started?

**Comment by aptalca** [comment]
We don't provide or suggest this anywhere
`/storage/.kodi/userdata/addon_data/service.system.docker/docker/volumes/doublecommander:/config`

Use a different folder like `/storage/doublecommander`

**Comment by vdb86** [comment]
Docker compose says:
path/to/config:/config

So I've put my path to config as I use with other containers as well.

I'll try with your suggestion, should that ensure the setting is kept after an updated container runs?

**Comment by vdb86** [comment]
I've changed the path to storage/docker/doublecommander
But I'm still not seeing the settings files anywhere.
Do you know where they are saved?

**Comment by vdb86** [comment]
After a very detailed review and a lot of help on Discord multiple things were caught:
- The first issue is that I'm using PUID=0, PGID=0 - these values are wrong and are making the container save the settings in a wrong folder /root/.config/doublecmd/ instead of /config/.config/doublecmd/
- After we figured that out I tried using the proper values for libreelec host (PUID=65534, PGID=100), but it still didn't work. After checking out the logs we saw that PGID was always the same as PUID, that's because I'm using Portainer and it doesn't allow you to set these variables as different values.

I'm 100% sure that if you don't use Portainer, but actually create your containers directly through docker everything will work just fine.

Big thanks for linuxserver for awesome images and for the discord people who showed a lot of patience and good will throughout this investigation.
